pages/app/site/[id]/settings.tsx

```diff
@@ -12,12 +12,22 @@ import toast, { Toaster } from "react-hot-toast";
 import { useDebounce } from "use-debounce";
 import { fetcher } from "@/lib/fetcher"
 
+type siteData = {
```

Comment:
Usually prefer use of `interface`'s here ([Reason](https://twitter.com/jaredpalmer/status/1331695529470922754?s=20))
```suggestion
interface SiteData {
```